addons/product/models/product_attribute_value.py

```diff
@@ -98,26 +93,35 @@ def write(self, values):
             self.env.invalidate_all()
         return res
 
-    @api.ondelete(at_uninstall=False)
-    def _unlink_except_used_on_product(self):
+    def get_error_message(self):
```

Comment:
maybe we can choose a better name for the function because it is doing more than getting an error message, smth like `check_pavs_are_used` or you can come up with smth better :)